### DISCUSSION TITLE
Project still active? v2 release?

### DISCUSSION THREAD

**Discussion by michaelGregoire** [Akryum/v-tooltip#946] [General]
Hello Floating Vue team. I'm noticing that there haven't been any commits in main in the past 3 months. Is the project still active? If so, when can we expect the final release of v2?

**Comment by AnthonyRuelle** [comment]
Up..


What’s going on with this project? Is he abandoned? Still no view on the stable version of v2 :(

**Comment by Akryum** [comment]
Stable version is out
